pilot/pkg/kube/inject/mesh.go

```diff
@@ -39,7 +39,7 @@ initContainers:
     capabilities:
       add:
       - NET_ADMIN
-    [[ if eq .DebugMode true -]]  
+    [[ if eq .DebugMode true -]]
```

Comment:
I'm not sure we need this file - we can add a template to the release (as file), for people who want to customize it, but the default should be to fetch from the cluster as before.
Not a problem for this PR, but we should discuss it.

Reply:
> default should be to fetch from the cluster as before

The default has always been to use hardcoded parameters :) It's easy enough to change, but may cause backwards compat. problems for users who have baked `kube-inject` into their CI/CD piplines. cc @linsun @christian-posta